pandas/_libs/tslibs/conversion.pyx

```diff
@@ -216,6 +216,11 @@ cdef class _TSObject:
     #    npy_datetimestruct dts      # npy_datetimestruct
     #    int64_t value               # numpy dt64
     #    object tzinfo
+    #    bint fold, default 0
+
+    # Set fold to 0 by default
```

Comment:
hmm, we don't do this with other attributes, just document them here.

Reply:
I think its ok to not do this here and just always set it explicity

Reply:
Done. I don't have a preference here: assigning it explicitly looks tidier to me, but using `__cinit__` looks a bit more robust (even as it makes catching errors more difficult).